<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeStaticMeshEditorWindow</class>
 <widget class="QWidget" name="WeStaticMeshEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Static Mesh Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
  <item>
    <widget class="QToolBar" name="toolBar">
     <property name="windowTitle">
      <string>toolBar</string>
     </property>
     <property name="allowedAreas">
      <set>Qt::AllToolBarAreas</set>
     </property>
     <property name="iconSize">
      <size>
       <width>19</width>
       <height>19</height>
      </size>
     </property>
     <property name="floatable">
      <bool>true</bool>
     </property>
     <addaction name="separator"/>
     <addaction name="actionReimport"/>
     <addaction name="separator"/>
    </widget>
   </item>
   <item>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="viewportFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="WeViewportWidget" name="viewportPreview" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_numVerteces">
          <property name="text">
           <string>Number of Verteces:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_numVertecesValue">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>%Value%</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_numTriangles">
          <property name="text">
           <string>Number of Triangles:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_numTrianglesValue">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>%Value%</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_resourceSize">
          <property name="text">
           <string>Resource Size:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_resourceSizeValue">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>%Value%</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  </item>
  </layout>
  <action name="actionReimport">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Reimport</string>
   </property>
   <property name="toolTip">
    <string>Reimport Asset</string>
   </property>
  </action>
  <action name="actionR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>R</string>
   </property>
   <property name="toolTip">
    <string>Toggle Red Channel</string>
   </property>
  </action>
  <action name="actionG">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>G</string>
   </property>
   <property name="toolTip">
    <string>Toggle Green Channel</string>
   </property>
  </action>
  <action name="actionB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="toolTip">
    <string>Toggle Blue Channel</string>
   </property>
  </action>
  <action name="actionA">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="toolTip">
    <string>Toggle Alpha Channel</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WeViewportWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/ViewportWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
